### DISCUSSION TITLE
Documentation for instance events

### DISCUSSION THREAD

**Discussion by btous** [nhn/tui.calendar#1313] [General]
Hi, 

I have been trying to get a list of all the instance events that can be listened from the calendar instance and how to listen them when they trigger.

After some hours I managed to found it in the code implementation (not the better place to go) and finally I got them here:
[https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/calendar.md#instance-events](https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/calendar.md#instance-events)

I think that it would be nice to add this info also in this documentation page for other users like me:
[https://nhn.github.io/tui.calendar/latest/](https://nhn.github.io/tui.calendar/latest/)

**Comment by adhrinae** [comment]
Sorry to hear that. We moved most of the documentation to markdown files for better readability. and we are leaving the docs website(https://nhn.github.io/tui.calendar/latest/) to provide examples.

I admit that the docs website needs clearer instructions guiding users to look at markdown files.

**Reply by btous** [reply]
Ok, I see that. So yes, probably avoiding duplicity and using the docs website only to provide examples will be better to prevent users like me thinking they are in the right place to see all the documentation and missing important parts only explained in the markdown.

Thanks for the answer :)
